src/components/collection-list/CollectionList.tsx

```diff
@@ -0,0 +1,37 @@
+import { UserCollection } from 'audius-client/src/common/models/Collection'
+import { Text } from 'react-native'
+
+import { CardList, CardListProps } from 'app/components/core'
+import { CollectionCard } from 'app/screens/explore-screen/components/CollectionCard'
+
+import { EmptyTab } from '../../screens/profile-screen/EmptyTab'
```

Comment:
Should we move/rename the EmptyTab component?

Reply:
yeah def! will rename to EmptyListItem i guess?